clicache/src/impl/PdxLocalWriter.cpp

```diff
@@ -179,23 +179,29 @@ namespace Apache
             return this;
           }
 
-          IPdxWriter^ PdxLocalWriter::WriteByte( String^ fieldName, SByte value )
+          IPdxWriter^ PdxLocalWriter::WriteUnsignedByte( String^ fieldName, Byte value )
```

Comment:
this file appears to be indented one space, instead of two?  Except where it contains tabs, of course.  Basically it looks all messed up, and we should probably pick a style and make it consistent.

Actually I guess it is two, but let's lose the tabs and whatever ails line 353.  Also pls resolve/remove your TODO comment on line 188

Reply:
Fixed.